components/compose/awesomebar/src/main/java/mozilla/components/compose/browser/awesomebar/internal/Suggestion.kt

```diff
@@ -66,7 +66,7 @@ internal fun Suggestion(
         )
         if (suggestion.editSuggestion != null) {
             AutocompleteButton(
-                onAutoComplete = { onAutoComplete(suggestion) },
+                onAutoComplete = { onAutoComplete() },
```

Comment:
nit: would just making it `onAutoComplete = onAutoComplete` work?

Reply:
Yeah, you are right! I'll update this.